apps/lib/http_server.c

```diff
@@ -266,19 +271,19 @@ int http_server_get_asn1_req(const ASN1_ITEM *it, ASN1_VALUE **preq,
     char reqbuf[2048], inbuf[2048];
     char *meth, *url, *end;
     ASN1_VALUE *req;
-    int ret = 1;
+    int ret = 0;
 
     *preq = NULL;
     if (ppath != NULL)
         *ppath = NULL;
     *pcbio = NULL;
 
-    /* Connection loss before accept() is routine, ignore silently */
+    log_message(prog, LOG_DEBUG, "Awaiting next request...");
+/* Connection loss before accept() is routine, ignore silently */
```

Comment:
Why the indentation change?

Reply:
This indentation change was by mistake.
It will be changed (with correct indentation) by #15053.